src/LibraryManager/Providers/Unpkg/UnpkgProvider.cs

```diff
@@ -22,7 +22,7 @@ public UnpkgProvider(IHostInteraction hostInteraction)
 
         public string Id => IdText;
 
-        public string NuGetPackageId => null;
+        public string NuGetPackageId { get; } = "Microsoft.Web.LibraryManager.Build";
```

Comment:
You could've just left the =>, but it doesn't really matter.

Reply:
Why do providers know anything about the NuGet package for restore?  This seems like a broken design.  Should this be fixed in the command instead?

Reply:
I think this is good  Preview3 as the simplest fix, but in general I completely agree.

Reply:
Can't we just change the `packageIds` line in RestoreOnBuildCommand.cs to be "Microsoft.Web.LibraryManager.Build"?

Reply:
oh, but currently this allows a provider to specify another package... hm, maybe not then.

Reply:
I would just leave as is for now. Not sure what was the intent of the original design. We can propose design changes later since this fix if for preview3

Reply:
Logged issue #155 for the design review